router.go

```diff
@@ -19,7 +20,7 @@ func NewRoute() *Route {
 	}
 }
 
-func (r *Route) Call(req *http.Request) interface{} {
+func (r *Route) Call(req *http.Request) (interface{}, error) {
```

Comment:
I think to refactor `Call` to `Handle`, send `http.ResponseWriter` into Handle, stop return `interface{}`
What do you say?

Reply:
let router handle response is little weird for me.

Reply:
Ok, that's fine. Seems NotFound is handling by Rocket, it would be strange exactly.